custom_components/tuya_local/devices/medion_life_p1002.yaml

```diff
@@ -0,0 +1,81 @@
+name: Portable Air Conditioner
+products:
+  - id: tqgodaa9suvtbflg
+    name: MEDION AIR Conditioner P1002
+primary_entity:
+  entity: climate
+  icon: "mdi:air-conditioner"
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: false
+          value: "off"
+        - dps_val: true
+          constraint: mode
+          conditions:
+            - dps_val: Cool
+              value: cool
+            - dps_val: Dyr
+              value: dry
+            - dps_val: Fan
+              value: fan_only
+    - id: 2
+      type: integer
+      name: temperature
+      range:
+        min: 16
+        max: 32
+      unit: C
+    - id: 3
+      type: integer
+      name: current_temperature
+    - id: 4
+      type: string
+      name: mode
+      hidden: true
+    - id: 5
+      type: string
+      name: fan_mode
+      mapping:
+        - dps_val: Low
+          value: low
+        - dps_val: High
+          value: high
+    - id: 22
```

Comment:
Move this to the problem binary_sensor below, as it will be more convenient/obvious to retrieve the fault_code from the same sensor that is indicating a fault.